src/index.js

```diff
@@ -109,12 +118,13 @@ export default class List extends React.Component {
 List.defaultProps = {
   source: [],
   rowHeight: 24,
-  overScanCount: 5,
+  overScanCount: 5
 };
 
 List.propTypes = {
   renderItem: PropTypes.func,
   rowHeight: PropTypes.number,
   source: PropTypes.array.isRequired,
   overScanCount: PropTypes.number.isRequired,
+  className: PropTypes.string
```

Comment:
Length rule requirement here.

Reply:
Order the proptypes based on the length of the line in ascending order. So the `className` prop needs to be third in this list. This is just a code formatting preference and doesn't affect the functionality of the code.